packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -11,6 +11,18 @@
 </section>
 <section>
   <h3 class="dummy-h3" id="section-component-api">Component API</h3>
+  <dl class="dummy-component-props">
+    <dt>listPosition <code>string</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>left</li>
+        <li class="default">right</li>
+      </ol>
+      <p>If no <span class="dummy-code">listPostion</span> is defined, it will automatically be positioned to the right.</p>
```

Comment:
This will go in the "How to use" section. Here it is already implicit by the fact that `right` is the _default_ value.

```suggestion
```

Reply:
I'll remove these in a separate PR; right now all of the API definitions have a sentence like this. 👍